pitch.md

```diff
@@ -23,38 +58,137 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+
+I recently found some simulators for poi, which I consider the same domain as 
+glowsticking. 
+
+[Flower Simulator](http://www.rationalpeople.us/double.html) is a great tool for 
+visualizing a specific move (flower) in 2D.
+
+[VisualSpinner3D](http://www.spinmorepoi.com/theory/) 
+looks promising, but it doesn't show on my computer :(. Critique group: does it 
+work on your computer?
+
+Both simulators are cool, but they are more on the systems side rather than the 
+language side because they aren't very composable. You need to give a name for a 
+trick instead of my idea: connecting motions to create a trick. Moreover, the 
+tricks available are limited to what has already been premade/hard-coded. But 
+they are good at animating individual moves.
+
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+As long as I focus on the composition of glowsticking moves, I will spend a lot 
+of time on language design. My syntax isn't finalized, since there may be better 
+ways to describe circular movements. So figuring out the syntax and semantics 
+will take up a lot of time.
+
+Ideally, I would try to connect my DSL to the poi simulators, which may involve 
+lots of research on the systems. Then again, I'm also creating an internal 
+representation for the poi simulators, so that's language design.
+
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think describing glowsticking/poi moves is quite ambitious. It's not as simple 
+as juggling or dance notation, which have simpler trajectories and lack complex 
+object manipulation.
+
+But after finding (open source) poi simulators, my goal is much clearer: using 
+my DSL to translate simple motions into glowsticking moves. I think having this 
+goal limits my scope in a good way so that I can focus on one simulator system.
+
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This is a good project because I can create a grammar for something that cannot 
+be explained well currently. Also, I have access to poi simulators and their 
+source code, so I can create internal representations that translate my DSL to 
+functional input in the poi simulators.
+
+Some challenges might include deciding which properties of circular motions are 
+really important in describing glowsticking moves. In addition, working with the 
+simulators might be challenging (I haven't looked at the source code yet).
+
+I'm not sure how this project will go off track because it has a specific goal 
```

Comment:
It seems like you could potentially spend a lot of time trying to figure out the systems aspect, but it seems like you have a good idea of how to simplify the systems aspect to focus on the language design!